OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/IUserManager.kt

```diff
@@ -138,4 +138,15 @@ interface IUserManager {
      * Return a copy of all local tags from the current user.
      */
     fun getTags(): Map<String, String>
+
+    /**
```

Comment:
Getters for onesignal ID and external ID needs to be exposed through this interface for public consumption